apps/src/templates/certificates/StudentsBeyondHoc.jsx

```diff
@@ -152,8 +153,8 @@ class StudentsBeyondHoc extends Component {
         <h1 style={headingStyle}>{heading}</h1>
         <VerticalImageResourceCardRow cards={cards} />
         {isEnglish && (
-          <CourseBlocksStudentGradeBands
-            showContainer={false}
```

Comment:
what happened to this prop?

Reply:
oh nevermind I see -- you pulled that logic out into its own props that aren't needed here 👍

Reply:
Yup - it basically controlled whether or not text was displayed, so now we can either send that text (heading, descriptions, etc.) through in the `CourseBlocksWrapper` props or, like in this case, just send nothing.